<launch>

  <arg name="UAV_NAME" default="$(env UAV_NAME)"/>
  <arg name="custom_config" default="" />

  <group ns="$(arg UAV_NAME)">

    <node name="TrajectoryRandomFlier" pkg="nodelet" type="nodelet" args="standalone mrs_testing/TrajectoryRandomFlier" output="screen" >

      <rosparam file="$(find mrs_testing)/config/trajectory_random_flier.yaml" />

      <!-- Load a user param file -->
      <rosparam if="$(eval not arg('custom_config') == '')" file="$(arg custom_config)" />

      <param name="uav_name" value="$(arg UAV_NAME)" />

        <!-- Subscribers -->
      <remap from="~position_command_in" to="control_manager/position_cmd" />

        <!-- Services -->
      <remap from="~activate_in" to="~activate" />
      <remap from="~trajectory_reference_out" to="control_manager/trajectory_reference" />

    </node>

  </group>
</launch>
